apps/frontend/src/components/global/Sidebar.vue

```diff
@@ -1,37 +1,103 @@
-        :all-selected="all_profiles_selected"
-        @toggle-all="toggle_all_profiles"
-        @changed-files="$emit('changed-files')"
-      />
-    </v-expansion-panels>
-  </v-navigation-drawer>
+  <div>
+    <ChecklistTargetDataModal
+      :visible="showTargetModal"
+      @close-modal="showTargetModal = false"
+    />
+    <ChecklistTechnologyAreaModal
+      :visible="showTechnologyModal"
+      @close-modal="showTechnologyModal = false"
+    />
+
+    <!-- Due to how the vuetify components work, one drawer is permanent to always be appended to the side and the other will be the temporary that can be pulled out -->
+    <v-navigation-drawer
+      :clipped="$vuetify.breakpoint.lgAndUp"
+      app
+      :style="{'z-index': 11}"
+      permanent
+      width="45px"
+      @input="$emit('input', $event)"
+    >
+      <v-container
+        v-if="!isUtilityDrawerShown"
+        fill-height
+        fluid
+        class="clickable"
+        @click="isUtilityDrawerShown = !isUtilityDrawerShown"
+      >
+        <v-row align="center" justify="center">
+          <v-col>
+            <v-icon>mdi-arrow-right</v-icon>
+          </v-col>
+        </v-row>
+      </v-container>
+    </v-navigation-drawer>
+    <v-navigation-drawer
+      v-model="isUtilityDrawerShown"
+      :clipped="$vuetify.breakpoint.lgAndUp"
+      app
+      :style="{'z-index': 11, 'margin-top': classification ? '5em' : '3em'}"
+      temporary
+      width="600px"
+      @input="$emit('input', $event)"
+    >
+      <div v-if="isUtilityDrawerShown">
+        <v-expansion-panels v-model="active_path" accordion>
+          <DropdownContent
+            header-text="Results"
+            :files="visible_evaluation_files"
+            :all-selected="all_evaluations_selected"
+            :enable-compare-view="true"
+            :compare-view-active="compareViewActive"
+            @toggle-all="toggle_all_evaluations"
+            @toggle-compare-view="compareView"
+            @changed-files="$emit('changed-files')"
+          />
+          <DropdownContent
+            header-text="Profiles"
+            :files="visible_profile_files"
+            :all-selected="all_profiles_selected"
+            @toggle-all="toggle_all_profiles"
+            @changed-files="$emit('changed-files')"
+          />
+          <DropdownContent
+            header-text="Checklists"
+            :files="visible_checklist_files"
+            @changed-files="$emit('changed-files')"
+          />
+        </v-expansion-panels>
+        <div class="mx-5 mr-10 mb-5">
+          <!-- Checklist Data Modals -->
+          <v-row v-if="inChecklistView" class="my-4" justify="center">
+            <v-btn
+              id="target-data-btn"
+              class="mx-2"
+              @click="setShowTargetModal"
+            >
+              <span class="d-none d-md-inline pr-2">
+                Add/Update Target Data
+              </span>
+            </v-btn>
+            <v-btn
```

Comment:
for all of these buttons, can you make it so that they don't hit the worst case width behavior that soon (i.e. remove the labels on themselves when they're still full size)

Reply:
Fixed. Labels aren't removed anymore